### ISSUE TITLE
[wasm] tf.keras.layers.Dense layer is not supported

### ISSUE THREAD

**Issue by tomerip** [tensorflow/tfjs#3265]
#### TensorFlow.js version
1.7.4

#### Describe the problem or feature request
Using a dense layer in a tf.keras model (tested on TF 2.1.0) results in a 'prod' op in the TFJS graph model after conversion (converter version 1.7.4).
It runs fine on GPU and CPU with backends that aren't WASM, but on WASM it gives:
`'prod' not yet implemented or not found in the registry. Did you forget to import the kernel?`

I managed to overcome the issue by using
`tf.keras.layers.Conv2D(output_size, kernel_size=(1,1), strides=(1,1), padding='same')`
instead of
`tf.keras.layers.Dense(output_size)`

After expanding dims to load the weights from Dense layers to the Conv2D layers, they produce exactly the same results.
Then, converting the model to TFJS doesn't produce the 'prod' op anymore and it works on WASM backend.

#### Code to reproduce the bug / link to feature request
```
import tensorflow as tf    
import tensorflow.keras.layers as layers


class ExampleModel(tf.keras.Model):
    def __init__(self):
        
        super().__init__()
        self.layer1 = layers.Dense(10)
        self.layer2 = layers.Conv2D(10, kernel_size=(1,1), strides=(1,1), padding='same')
        
    def call(self, x):
        x = self.layer1(x) # won't work
        x = self.layer2(x) # will work
        return x

model = ExampleModel()

model.save('/path/to/SavedModel', format='tf')
```

Converting the SavedModel using tfjs converter results in said issue.

Is there something I'm missing? I feel like tf.keras is very popular after 2.0.0 and a Dense layer is one of the basics, yet I was surprised to find out no one else was complaining about it.

**Comment by annxingyuan** [comment]
@rthadur This should be fixed on latest version of TFJS which includes prod for WASM.

**Comment by rthadur** [comment]
Yes Ann, this was added here https://github.com/tensorflow/tfjs/pull/4138 , Thank you.Closing this request.
